src/components/ArtistProfiles.res

```diff
@@ -171,7 +171,9 @@ module ArtistDetails = {
           }}
         </Rimble.Box>
         <Rimble.Box p=1 width=[1., 1., 0.3333]>
-          <h2> {"Total contributed by artist"->React.string} </h2>
+          <h2> {"Total contributed through artwork created by "->React.string} {optName
+                ->Option.getWithDefault(Option.getWithDefault(optArtistName, "Loading artist name"))
+                ->React.string}</h2>
```

Comment:
```suggestion
          <h2> {"Total generated by "->React.string} {optName
                ->Option.getWithDefault(Option.getWithDefault(optArtistName, "Loading artist name"))
                ->React.string} {"'s artworks"->React.string}</h2>
```

I think it was too verbose before. This also doesn't take into account people who have names that end in an s for plurals. But it is ok for now, something like this could help though: https://www.npmjs.com/package//pluralize